_projects/2024/100419840/100419840.Rmd

```diff
@@ -0,0 +1,598 @@
+---
+title: "How housing is no longer a right"
+description: Visualization of the housing crisis in Spain according to official data from the Ministry of Transportation.
+categories: "2024"
+author: Maria Carda Gargallo
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false 
+    toc: true
+---
+
+
+## Introduction
+
+The original graph originates from the article published by elDiario.es on the occasion of a demonstration on rental prices in Madrid last October 2024 (["El aumento del precio del alquiler y la especulación avivan las protestas por el derecho a la vivienda"](https://www.eldiario.es/economia/aumento-precio-alquiler-especulacion-avivan-protestas-derecho-vivienda_1_11726321.html)). This article highlights different aspects of this housing crisis, and shows the boom in home purchase prices despite the slight increase in wages in recent years.
+
+
+### The original graph
+
+The graph shows the *Quarterly evolution of the appraised value of free housing per quarter in each Autonomous Community with respect to the quarter where the price of housing peaked in each community*, i.e. the variation in the appraised price of housing with respect to the 2008 crisis. It highlights the values of three communities (Madrid, Baleares and Canarias), which are those with the fastest growth trends in recent months.
+
+![Original graph](https://raw.githubusercontent.com/mariacardag/datavizclass/main/image.png)
```

Comment:
Please set it as external as e.g. David does [here](https://github.com/IBiDat/dataviz/blob/9a3bc0367b1c0b297765b59ff6f3c34cef06a204/_projects/2024/100535712/100535712.Rmd#L26).